packages/zoe/src/contractSupport/interest.js

```diff
@@ -0,0 +1,55 @@
+// @ts-check
+
+import {
+  floorDivideBy,
+  floorMultiplyBy,
+  invertRatio,
+  multiplyRatios,
+} from './ratio.js';
+
+/**
+ *
+ * @param {Ratio} currentCompoundedInterest
+ * @param {Ratio} previousCompoundedInterest
+ * @returns {Ratio} additional compounding since the previous
+ */
+export const calculateRelativeCompounding = (
```

Comment:
does this need to be exported?

Reply:
not strictly. I thought it might be useful but I'll unexport and leave that to when someone needs it.

Reply:
My thought was more that if it's exported, then it needs to be tested separately. If it's only used internally, then testing the external method that uses it should be sufficient.

If you expect it to be separately useful, it would be helpful to add tests for it.